client-js/app/scripts/places.ts

```diff
@@ -1,19 +1,35 @@
 // Function for creating URLs of all the places in the application.
 
-export function grapherUrl(path: Array<string>, dir?: string): string {
-  return '/grapher/' + pathStr(path, dir);
-}
+export class Places {
+  fullPrefix: string;
```

Comment:
So I understand the functions in here for creating URLs but I don't know what the ```Places``` class is or how it helps? I can see it invokes as a property in all the sub-applications but I don't undestand what's it doing. What's required beyond just taking a path to a resource and generating a URL?